concepts.wip/boolean-logic/introduction.md

```diff
@@ -1,57 +1,62 @@
 # Introduction
 
-Imagine we have the following Boolean expressions in Julia: `5 > 3` and `1 != 0`, both of the expression are logically `true`. How can we combine the previously mentioned expressions to produce a `true` or `false` result?
-
-We use [logical connectives](https://en.wikipedia.org/wiki/Logical_connective)! Also called _logical operators_. In Julia, they are called [Boolean operators](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Boolean-Operators).
+In Julia, according to the [operator precedence](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Operator-Precedence-and-Associativity), `&&` has higher prority over `||`.
 
-Below, we provide examples on how to use them to produce the desired result.
+Imagine we have the following Boolean expressions in Julia: `5 > x` and `x != 0`.
+If `x` was 3 they would both be `true`.
+We can express statements like "is x less than 5 and not equal to y?" using [Boolean operators](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Boolean-Operators): `!` (not), `&&` (and), `||` (or).
 
-## Logical _and_
+### Logical _not_
 
 ```julia
-julia> 5 > 3
+# ! - represents logical not in  Julia.
+# Also called negation.
```

Comment:
I think these should be moved outside the code to the text, and the codeblocks should be `julia-repl`, not `julia`.

Reply:
Moving the comments outside is good, but for consistency we'd need to do the same for the "and" and "or" examples.